rs/sns/testing/BUILD.bazel

```diff
@@ -97,9 +99,20 @@ DEV_ENV = {
 }
 
 rust_binary(
-    name = "cli",
+    name = "sns-testing",
     testonly = True,
-    srcs = ["src/main.rs"],
+    srcs = ["src/bin/main.rs"],
+    data = DEV_DATA,
+    env = DEV_ENV,
+    deps = DEPENDENCIES + [
+        ":sns_testing",
+    ],
+)
+
+rust_binary(
```

Comment:
I wonder why we need this as a separate binary, couldn't it be a subcommand of `sns-testing`?

Reply:
My motivation was to unclutter the subcommand interface. Additionally, we have more-or-less logical separation: `sns-testing-init` works with PocketIC API, while `sns-testing` works with `ic-agent` API